README.md

```diff
@@ -24,3 +24,76 @@ fn main() -> Result<(), FormulaError> {
     Ok(())
 }
 ```
+
```

Comment:
Can you please add replace line with this in the introduction?

"# frequenz-microgrid-formula-engine-rs
A library to create formulas over streamed data, primarily used for calculating and processing values within microgrid applications."

Reply:
Please also mention at the top of this section that the formula engine can only work on _resampled_ component data streams.

Maybe also that it has been designed to work with 
- our resampling library (which sends None values when there is no data from a component, for example): https://github.com/frequenz-floss/frequenz-resampling-rs, and 
- our component graph library for generating formulas: https://github.com/frequenz-floss/frequenz-microgrid-component-graph-rs